Création Leclerv2 et rajout MarcheConsommateurs partiel

MarcSuteau opened the pull request.

Commit:
inutile

Commit:
Merge branch 'master' of https://github.com/MarcSuteau/CACAO2016.git

Commit:
Merge branch 'master' of https://github.com/MarcSuteau/CACAO2016.git

Conflicts:
	PRIME_CACAO_2016/src/abstraction/equipe3/Leclerc.java

Commit:
Merge branch 'master' of https://github.com/MarcSuteau/CACAO2016.git

MarcSuteau committed:
correction du next pour avoir les bonnes formules du solde

Commit:
essai

Commit:
Merge branch 'master' of https://github.com/MarcSuteau/CACAO2016.git

MarcSuteau committed:
rectification de l'indicateur solde en rajoutant une fonction

Commit:
essai

Commit:
Merge branch 'master' of https://github.com/MarcSuteau/CACAO2016.git

Conflicts:
	PRIME_CACAO_2016/src/abstraction/equipe3/Leclerc.java

Commit:
s

Commit:
essai

Commit:
Refonte du côde pour avoir un nombre de transformateur infini

Commit:
Cadeau

Tommy56 committed:
try

Commit:
esssai

MarcSuteau committed:
f

MarcSuteau committed:
Merge remote-tracking branch 'origin/Mael'

Conflicts:
	PRIME_CACAO_2016/bin/.gitignore
	PRIME_CACAO_2016/src/abstraction/equipe3/Leclerc.java

Tommy56 committed:
Merge branch 'master' of https://github.com/MarcSuteau/CACAO2016.git

Conflicts:
	PRIME_CACAO_2016/src/abstraction/equipe3/Leclerc.java

Commit:
modification méthode next

Tommy56 committed:
Merge remote-tracking branch 'origin/Mael'

Conflicts:
	PRIME_CACAO_2016/bin/.gitignore
	PRIME_CACAO_2016/src/abstraction/equipe3/Leclerc.java

MarcSuteau committed:
a

Commit:
modification ratio

MarcSuteau committed:
Merge remote-tracking branch 'choose_remote_name/master'

Conflicts:
	PRIME_CACAO_2016/bin/.gitignore
	PRIME_CACAO_2016/src/abstraction/commun/IDistributeur.java
	PRIME_CACAO_2016/src/abstraction/commun/MondeV1.java

Commit:
.

MarcSuteau committed:
Merge branch 'master' of https://github.com/MarcSuteau/CACAO2016.git

Conflicts:
	PRIME_CACAO_2016/src/abstraction/equipe3/Leclerc.java

Commit:
Class Stock

Commit:
Merge remote-tracking branch 'origin/master' into Mael2

Commit:
modification class stock

Commit:
modification class Leclerc

MarcSuteau committed:
Merge remote-tracking branch 'choose_remote_name/master'

Conflicts:
	PRIME_CACAO_2016/bin/.gitignore

Commit:
modification stock dans class Leclerc

MarcSuteau committed:
m

MarcSuteau committed:
Merge remote-tracking branch 'origin/Mael2'

Conflicts:
	PRIME_CACAO_2016/src/abstraction/equipe3/Leclerc.java

Commit:
modif class Stock pour avoir 3 produits

Tommy56 committed:
thomas

Tommy56 committed:
Merge remote-tracking branch 'origin/Mael2'

Conflicts:
	PRIME_CACAO_2016/bin/.gitignore

Commit:
modif class Commande

MarcSuteau committed:
Merge remote-tracking branch 'choose_remote_name/master'

Conflicts:
	PRIME_CACAO_2016/src/abstraction/commun/ITransformateur.java

Tommy56 committed:
Merge branch 'master' of https://github.com/MarcSuteau/CACAO2016.git

Commit:
début modification class Leclerc

Commit:
Merge remote-tracking branch 'origin/master' into Mael2

MarcSuteau committed:
Merge remote-tracking branch 'choose_remote_name/master'

Conflicts:
	PRIME_CACAO_2016/src/abstraction/commun/IDistributeur.java
	PRIME_CACAO_2016/src/abstraction/equipe3/Leclerc.java
	PRIME_CACAO_2016/src/abstraction/equipe6/Carrefour.java

Commit:
implementation frais de stock

Commit:
Merge remote-tracking branch 'origin/master' into Mael2

Conflicts:
	PRIME_CACAO_2016/src/abstraction/equipe3/Leclerc.java

Commit:
Implementation de demande

MarcSuteau committed:
retrait des erreurs

MarcSuteau committed:
Merge remote-tracking branch 'choose_remote_name/master'

Conflicts:
	PRIME_CACAO_2016/bin/.gitignore
	PRIME_CACAO_2016/src/abstraction/commun/IDistributeur.java
	PRIME_CACAO_2016/src/abstraction/equipe3/Leclerc.java
	PRIME_CACAO_2016/src/abstraction/equipe6/Carrefour.java

MarcSuteau committed:
rajout de corrections mineures

MarcSuteau committed:
s

Commit:
modif

Commit:
Merge remote-tracking branch 'origin/master' into Mael2

Conflicts:
	PRIME_CACAO_2016/src/abstraction/commun/IDistributeur.java
	PRIME_CACAO_2016/src/abstraction/equipe3/Leclerc.java

Tommy56 committed:
Merge branch 'master' of https://github.com/MarcSuteau/CACAO2016.git

Conflicts:
	PRIME_CACAO_2016/src/abstraction/commun/IDistributeur.java

Tommy56 committed:
Merge branch 'master' of https://github.com/MarcSuteau/CACAO2016.git

Conflicts:
	PRIME_CACAO_2016/bin/.gitignore
	PRIME_CACAO_2016/src/abstraction/commun/IDistributeur.java

Tommy56 committed:
Merge remote-tracking branch 'origin/Mael2'

Tommy56 committed:
Implementation du prix de vente

MarcSuteau committed:
raji=out classe leclercv2 pour les modif, on re changera apres

MarcSuteau committed:
Merge branch 'master' of https://github.com/MarcSuteau/CACAO2016.git

MarcSuteau committed:
b

MarcSuteau committed:
pas mal de changements notamment sur leclercv2

Commit:
fdsf

Commit:
Merge remote-tracking branch 'origin/master' into Mael2

Conflicts:
	PRIME_CACAO_2016/src/abstraction/equipe3/Leclerc.java

MarcSuteau committed:
rajout de contredemande et quelques fonctions pour faciliter le code

MarcSuteau committed:
Merge remote-tracking branch 'choose_remote_name/master'

Conflicts:
	PRIME_CACAO_2016/bin/.gitignore

Commit:
Merge remote-tracking branch 'origin/master' into Mael2

Conflicts:
	PRIME_CACAO_2016/bin/.gitignore

felixp49 committed:
A

felixp49 committed:
Modification IDistributeur

Commit:
Implementation getStock et getPrixDeVente dans LeclercV2

MarcSuteau committed:
rectification de contre demande et déplacement de fonctions de
Lerclercv2 vers stock

MarcSuteau committed:
Merge remote-tracking branch 'origin/Mael2'

felixp49 committed:
Commit

felixp49 committed:
Commit marché

felixp49 committed:
Merge branch 'master' of https://github.com/MarcSuteau/CACAO2016.git

Conflicts:
	PRIME_CACAO_2016/src/abstraction/commun/IDistributeur.java

Commit:
modif Leclercv2

Commit:
Merge remote-tracking branch 'origin/master' into Mael2

Commit:
suppression class inutile

MarcSuteau committed:
x

MarcSuteau committed:
Merge remote-tracking branch 'origin/Mael2'

Commit:
ok

MarcSuteau committed:
d

MarcSuteau committed:
Merge remote-tracking branch 'origin/Mael2'

Conflicts:
	PRIME_CACAO_2016/src/abstraction/equipe3/Leclerc.java
	PRIME_CACAO_2016/src/abstraction/equipe3/Leclercv2.java
	PRIME_CACAO_2016/src/abstraction/equipe6/Carrefour.java

felixp49 committed:
Avancement

felixp49 committed:
kuguj

felixp49 committed:
A

felixp49 committed:
Merge branch 'master' of https://github.com/MarcSuteau/CACAO2016.git

Conflicts:
	PRIME_CACAO_2016/src/abstraction/equipe3/Leclercv2.java

felixp49 committed:
c

MarcSuteau committed:
c

Commit:
Thomas : mofid prix de vente

MarcSuteau committed:
Merge remote-tracking branch 'choose_remote_name/master'

Conflicts:
	PRIME_CACAO_2016/src/abstraction/commun/IDistributeur.java
	PRIME_CACAO_2016/src/abstraction/commun/ITransformateur.java
	PRIME_CACAO_2016/src/abstraction/equipe3/Leclerc.java
	PRIME_CACAO_2016/src/abstraction/equipe6/Carrefour.java

MarcSuteau committed:
Merge branch 'master' of https://github.com/MarcSuteau/CACAO2016.git

Conflicts:
	PRIME_CACAO_2016/src/abstraction/equipe3/Leclercv2.java

Commit:
fsd

Commit:
Merge remote-tracking branch 'origin/master' into Mael2

felixp49 committed:
Marchefinal

felixp49 committed:
fd

Commit:
erreur

Commit:
erreur

felixp49 committed:
Merge branch 'master' of https://github.com/MarcSuteau/CACAO2016.git

Commit:
modif marcheconso

felixp49 committed:
modif marche conso

Commit:
Merge remote-tracking branch 'origin/master' into Mael2

Conflicts:
	PRIME_CACAO_2016/src/abstraction/commun/MarcheConsommateurs.java

Commit:
modif recette

MarcSuteau committed:
d

MarcSuteau committed:
Merge remote-tracking branch 'origin/Mael2'

massimotisi commented:
Des autres pull request ont étés acceptés, il faut que vous resynchronisez.

MarcSuteau committed:
Merge remote-tracking branch 'choose_remote_name/master'

Conflicts:
	PRIME_CACAO_2016/src/abstraction/commun/MarcheProducteur.java

massimotisi commented:
- votre code lance une exception au demarrage
- J'ai besoin de l'OK explicit de @lakac
- J'espere que @florianed, @jhtr, @AlexandreMarty, @MarcSuteau, @SofianeSalort sont d'accord avec vos modifs au package commun

MarcSuteau commented:
Oui je suis en train de travailler sur ces erreurs, par contre le programme
fait une erreur lors de l'ajout des producteurs alors que nous n'avons à
priori pas toucher à leur code.

2016-05-31 11:34 GMT+02:00 Massimo Tisi notifications@github.com:

> - votre code lance une exception au demarrage
> - J'ai besoin de l'OK explicit de @lakac https://github.com/lakac
> - J'espere que @florianed https://github.com/florianed, @jhtr
>   https://github.com/jhtr, @AlexandreMarty
>   https://github.com/AlexandreMarty, @MarcSuteau
>   https://github.com/MarcSuteau, @SofianeSalort
>   https://github.com/SofianeSalort sont d'accord avec vos modifs au
>   package commun
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> https://github.com/CACAO2016/CACAO2016/pull/28#issuecomment-222639025,
> or mute the thread
> https://github.com/notifications/unsubscribe/AQ9gpX5iw8STbOf569_N6dknUEUL-KQLks5qHACTgaJpZM4Iokcm
> .

massimotisi commented:
Je ne vais pas accepter si le probleme de l'exception n'est pas resolu. Je voix plusieurs solutions alternatives: 1) vous discutez avec les producteurs et trouvez une solution commun, 2) vous n'appellez plus la methode des producteurs mais une alternative temporaire faite par vous, 3) vous ajoutez un try-catch pour capturer l'exception et imprimer un message en console.

MarcSuteau committed:
mise en commentaire des fonctions du marché Consommateurs qui
provoquaient des nullpointer, de même pour l'ajout des producteurs

MarcSuteau commented:
j'ai mis en commentaire l'appel des fonctions qui posaient problème au
niveau du marché consommateurs, je suis en train de voir avec les
producteurs pour la méthode peupler qui provoque des nullpointer exceptions
lors de l'ajout des producteurs, je les ai mises en commentaire en
attendant.

Le 31 mai 2016 à 13:07, Massimo Tisi notifications@github.com a écrit :

> Je ne vais pas accepter si le probleme de l'exception n'est pas resolu. Je
> voix plusieurs solutions alternatives: 1) vous discutez avec les
> producteurs et trouvez une solution commun, 2) vous n'appellez plus la
> methode des producteurs mais une alternative temporaire faite par vous, 3)
> vous ajoutez un try-catch pour capturer l'exception et imprimer un message
> en console.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> https://github.com/CACAO2016/CACAO2016/pull/28#issuecomment-222658121,
> or mute the thread
> https://github.com/notifications/unsubscribe/AQ9gpaRXeuWF2_2d99EBbrQIxki3PAGJks5qHBaEgaJpZM4Iokcm
> .

massimotisi commented:
J'attends que vous me dites que l'exception est disparue. 
Aussi svp annullez votre modification au fichier PRIME_CACAO_2016/src/abstraction/fourni/v0/Detaillant.java.

MarcSuteau committed:
retrait des modifications de v0 detaillant

MarcSuteau committed:
retrait de l'ajout du marche Consommateurs en tant qu'acteur car il
provoquait un nullpointer exception en attendant une correction pour que
le programme fonctionne et que les autres puissent avoir acces a notre
code. Une autre pull request réglera les problèmes et l'implementera
correctement.

MarcSuteau commented:
j'ai retiré le marche Consommateurs de la liste des acteurs, c'est lui qui
provoquait l'exception au final. Dans l'attente de pouvoir lui apporter une
correction, je l'ai mis en commentaire pour que vous puissiez accepter ma
pull request.

2016-05-31 14:11 GMT+02:00 Massimo Tisi notifications@github.com:

> J'attends que vous me dites que l'exception est disparue.
> Aussi svp annullez votre modification au fichier
> PRIME_CACAO_2016/src/abstraction/fourni/v0/Detaillant.java.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> https://github.com/CACAO2016/CACAO2016/pull/28#issuecomment-222670090,
> or mute the thread
> https://github.com/notifications/unsubscribe/AQ9gpctT75lWCNMWVHbNc1XhDcze4Mesks5qHCVjgaJpZM4Iokcm
> .

MarcSuteau committed:
modif ventes : rajout d'une fonction actualiser

massimotisi commented:
OK
